hetzner-kube cluster add-worker doesn't work

So i tried to add a worker in the version 0.3.2-SNAPSHOT and got that error.
I created the cluster in the same version.

```
user@pc:/some/path# hetzner-kube cluster add-worker --name sds -n 1 --worker-server-type cx21
Enter passphrase for SSH key /some/path/.ssh/id_rsa: 
2018/07/20 09:46:48 creating server 'sds-worker-02'...
  --- [======================================] 100%
2018/07/20 09:47:08 Created node 'sds-worker-02' with IP 159.69.53.71
panic: runtime error: slice bounds out of range

goroutine 1 [running]:
github.com/xetys/hetzner-kube/pkg/clustermanager.PrivateIPPrefix(0x0, 0x0, 0xc42014f560, 0x2)
	/some/path/go/src/github.com/xetys/hetzner-kube/pkg/clustermanager/wireguard.go:65 +0x12f
github.com/xetys/hetzner-kube/pkg/hetzner.(*Provider).CreateNodes(0xc4200dc480, 0x8ffc4a, 0x6, 0x0, 0x0, 0x7fff543896c3, 0x4, 0x0, 0x0, 0x0, ...)
	/some/path/go/src/github.com/xetys/hetzner-kube/pkg/hetzner/hetzner_provider.go:110 +0x675
github.com/xetys/hetzner-kube/pkg/hetzner.(*Provider).CreateWorkerNodes(0xc4200dc480, 0xc4201a3468, 0x7, 0x7fff543896c3, 0x4, 0xc4201b06f0, 0x3, 0x3, 0x1, 0x1, ...)
	/some/path/go/src/github.com/xetys/hetzner-kube/pkg/hetzner/hetzner_provider.go:146 +0x14b
github.com/xetys/hetzner-kube/cmd.glob..func5(0xbc1080, 0xc42007f3e0, 0x0, 0x6)
	/some/path/go/src/github.com/xetys/hetzner-kube/cmd/cluster_add_worker.go:102 +0x5e5
github.com/xetys/hetzner-kube/vendor/github.com/spf13/cobra.(*Command).execute(0xbc1080, 0xc42007f320, 0x6, 0x6, 0xbc1080, 0xc42007f320)
	/some/path/go/src/github.com/xetys/hetzner-kube/vendor/github.com/spf13/cobra/command.go:702 +0x2c6
github.com/xetys/hetzner-kube/vendor/github.com/spf13/cobra.(*Command).ExecuteC(0xbc34a0, 0xb7a9e0, 0x0, 0x0)
	/some/path/go/src/github.com/xetys/hetzner-kube/vendor/github.com/spf13/cobra/command.go:783 +0x2e4
github.com/xetys/hetzner-kube/vendor/github.com/spf13/cobra.(*Command).Execute(0xbc34a0, 0x8fe72e, 0x5)
	/some/path/go/src/github.com/xetys/hetzner-kube/vendor/github.com/spf13/cobra/command.go:736 +0x2b
github.com/xetys/hetzner-kube/cmd.Execute()
	/some/path/go/src/github.com/xetys/hetzner-kube/cmd/some/path.go:51 +0x7e
main.main()
	/some/path/go/src/github.com/xetys/hetzner-kube/main.go:20 +0x20

```
I'm operating on:
```
Distributor ID:	Ubuntu
Description:	Ubuntu 18.04 LTS
Release:	18.04
Codename:	bionic
```

I am getting the same error with 0.3.1-RELEASE (on mac) :(

```
hetzner-kube cluster add-worker --name test -n 1 --worker-server-type cx11
2018/08/03 11:38:49 creating server 'test-worker-02'...
  --- [======================================] 100%
2018/08/03 11:39:09 Created node 'test-worker-02' with IP 159.69.53.230
panic: runtime error: slice bounds out of range

goroutine 1 [running]:
github.com/xetys/hetzner-kube/pkg/clustermanager.PrivateIPPrefix(0x0, 0x0, 0x10bb9a64, 0x2)
	/home/david/go-path/src/github.com/xetys/hetzner-kube/pkg/clustermanager/wireguard.go:64 +0x81
github.com/xetys/hetzner-kube/pkg/hetzner.(*Provider).CreateNodes(0x10aeff80, 0x410c5b, 0x6, 0x0, 0x0, 0xbffffb8b, 0x4, 0x0, 0x0, 0x0, ...)
	/home/david/go-path/src/github.com/xetys/hetzner-kube/pkg/hetzner/hetzner_provider.go:110 +0x554
github.com/xetys/hetzner-kube/pkg/hetzner.(*Provider).CreateWorkerNodes(0x10aeff80, 0x10a17ec0, 0x15, 0xbffffb8b, 0x4, 0x10b2e6a0, 0x3, 0x3, 0x1, 0x1, ...)
	/home/david/go-path/src/github.com/xetys/hetzner-kube/pkg/hetzner/hetzner_provider.go:146 +0xc4
github.com/xetys/hetzner-kube/cmd.glob..func5(0x678940, 0x10a74ba0, 0x0, 0x6)
	/home/david/go-path/src/github.com/xetys/hetzner-kube/cmd/cluster_add_worker.go:102 +0x672
github.com/xetys/hetzner-kube/vendor/github.com/spf13/cobra.(*Command).execute(0x678940, 0x10a74b70, 0x6, 0x6, 0x678940, 0x10a74b70)
	/home/david/go-path/src/github.com/xetys/hetzner-kube/vendor/github.com/spf13/cobra/command.go:702 +0x1f2
github.com/xetys/hetzner-kube/vendor/github.com/spf13/cobra.(*Command).ExecuteC(0x679c60, 0x636fa0, 0x0, 0x0)
	/home/david/go-path/src/github.com/xetys/hetzner-kube/vendor/github.com/spf13/cobra/command.go:783 +0x26f
github.com/xetys/hetzner-kube/vendor/github.com/spf13/cobra.(*Command).Execute(0x679c60, 0x40f761, 0x5)
	/home/david/go-path/src/github.com/xetys/hetzner-kube/vendor/github.com/spf13/cobra/command.go:736 +0x1b
github.com/xetys/hetzner-kube/cmd.Execute()
	/home/david/go-path/src/github.com/xetys/hetzner-kube/cmd/root.go:51 +0x65
main.main()
	/home/david/go-path/src/github.com/xetys/hetzner-kube/main.go:20 +0x11
```

```
hetzner-kube version
0.3.1-RELEASE
```

--- a/pkg/clustermanager/cluster.go
+++ b/pkg/clustermanager/cluster.go
@@ -63,6 +63,7 @@ func (manager *Manager) Cluster() Cluster {
 		IsolatedEtcd:  manager.isolatedEtcd,
 		CloudInitFile: manager.cloudInitFile,
 		SelfHosted:    manager.selfHosted,
+		NodeCIDR:      manager.clusterProvider.GetNodeCidr(),
 	}
 }
 

--- a/pkg/clustermanager/interfaces.go
+++ b/pkg/clustermanager/interfaces.go
@@ -23,5 +23,6 @@ type ClusterProvider interface {
 	GetMasterNode() (*Node, error)
 	GetCluster() Cluster
 	GetAdditionalMasterInstallCommands() []NodeCommand
+	GetNodeCidr() string
 	MustWait() bool
 }

--- a/pkg/hetzner/hetzner_provider.go
+++ b/pkg/hetzner/hetzner_provider.go
@@ -222,6 +222,11 @@ func (provider *Provider) GetAdditionalMasterInstallCommands() []clustermanager.
 	return []clustermanager.NodeCommand{}
 }
 
+// GetNodeCidr returns the CIDR to use for nodes in cluster
+func (provider *Provider) GetNodeCidr() string {
+	return provider.nodeCidr
+}
+
 // MustWait returns true, if we have to wait after creation for some time
 func (provider *Provider) MustWait() bool {
 	return provider.wait

